compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -498,10 +499,69 @@ OMR::ARM64::TreeEvaluator::exceptionRangeFenceEvaluator(TR::Node *node, TR::Code
 
 TR::Register *
 OMR::ARM64::TreeEvaluator::loadaddrEvaluator(TR::Node *node, TR::CodeGenerator *cg)
-	{
-	// TODO:ARM64: Enable TR::TreeEvaluator::loadaddrEvaluator in compiler/aarch64/codegen/TreeEvaluatorTable.hpp when Implemented.
-	return OMR::ARM64::TreeEvaluator::unImpOpEvaluator(node, cg);
-	}
+   {
+   TR::Register *resultReg;
+   TR::Symbol *sym = node->getSymbol();
+   TR::Compilation *comp = cg->comp();
+   TR::MemoryReference *mref = new (cg->trHeapMemory()) TR::MemoryReference(node, node->getSymbolReference(), 4, cg);
```

Comment:
This `OMR::ARM64::MemoryReference` constructor is unimplemented as well.

Reply:
The `MemoryReference` constructor is implemented in #4122.

Reply:
Sorry.  I just saw that.

Reply:
I moved the `MemoryReference` constructor change from #4122 to this PR.